src/Shared/AWS/AWSSemanticConventions.Legacy.cs

```diff
@@ -62,6 +62,16 @@ private abstract class AWSSemanticConventionsLegacy : AWSSemanticConventionsBase
         public override string AttributeAWSBedrockGuardrailId => "aws.bedrock.guardrail.id";
         public override string AttributeAWSBedrockKnowledgeBaseId => "aws.bedrock.knowledge_base.id";
         public override string AttributeAWSBedrock => "aws_bedrock";
+        public override string AttributeAWSRegion => "aws.region";
+        public override string AttributeAWSRequestId => "aws.request_id";
+        public override string AttributeAWSSQSQueueName => "aws.sqs.queue_name";
+        public override string AttributeAWSS3BucketName => "aws.s3.bucket";
+        public override string AttributeAWSKinesisStreamName => "aws.kinesis.stream_name";
+        public override string AttributeAWSLambdaResourceMappingId => "aws.lambda.resource_mapping.id";
```

Comment:
for the attributes that are not yet part of the Semantic Convention - I'm not sure if they should be added as "Legacy".

I'm also not sure if they should be added to an existing code page - ie it feels odd to add them to say v.1.29.0 when they aren't actually part of that spec either.

**Any thoughts on this?**

OR -  it might be better to first send a PR to the Semantic Convention repo and get the attributes added - then they'll be part of a SemConv release and we wont have this problem.  Given they're attributes are all in the AWS namespace I would expect the PR to be accepted rather quickly.